tests/rados/test_config_parameter_chk.py

```diff
@@ -206,7 +212,28 @@ def run(ceph_cluster, **kw):
                                 f"In the mclock {profile}  {param[0]} parameter value is changeable"
                             )
                             return 1
-        return 0
+        # RocksDB compression parameter check
+
+    if config.get("scenario") == "rocksdb_compression":
+        # Checking the RHCS version.
+        if float(build) < 7.1:
+            log.info(
+                "Test running on version less than 7.1, skipping verifying rocksdb compression parameter check"
+            )
+            return 0
+
+        rocksdb_chk = mon_object.get_config(
+            section="osd", param="bluestore_rocksdb_options"
+        )
+        if "kLZ4Compression" not in rocksdb_chk:
+            log.error(
+                f"The default compression value is not  kLZ4Compression- {rocksdb_chk}"
+            )
+            return 1
+        log.info(
+            f"The RocksDB compression value is set to kLZ4Compression-{rocksdb_chk}"
```

Comment:
Please change this log line to say - `The default value of compression is found to be kLZ4Compression`
If we say `set to`, it might mislead someone to think that this value was set during the test execution.

Reply:
Modified the statement.